README.md

```diff
@@ -1,37 +1,105 @@
+  </a>
+</p>
 
-Requires Vue 2.0
+**Note**: This plugin is for Vue 3. [Check out the docs for Vue 2 version](https://github.com/storyblok/storyblok-vue/tree/master)
 
-## Install
+## 🚀 Usage
 
-```
-npm install storyblok-vue --save-dev
-```
+> If you are first-time user of the Storyblok, read the [Getting Started](https://www.storyblok.com/docs/guide/getting-started?utm_source=github.com&utm_medium=readme&utm_campaign=storyblok-vue) guide to get a project ready in less than 5 minutes.
 
-## Usage
+### Installation
 
-In your entry point:
+Install `@storyblok/vue@next`:
 
+```bash
+npm install --save-dev @storyblok/vue@next
+// yarn add -D @storyblok/vue@next
 ```
-import StoryblokVue from 'storyblok-vue'
 
-Vue.use(StoryblokVue)
+Register it globally on your application (usually in `main.js`):
+
+```js
+import { createApp } from "vue";
+import StoryblokVue from "@storyblok/vue";
+import App from "./App.vue";
+
+const app = createApp(App);
+app.use(StoryblokVue);
 ```
 
-In the template:
+#### From a CDN
+
+Install the file from the CDN:
+
+```html
+<script src="https://unpkg.com/@storyblok/vue@next"></script>
 ```
-<div v-editable="demo">
-  Test
-</div>
+
+### Getting started
+
+For every component you've defined in your Storyblok space, add the `v-editable` directive with the blok content:
+
+```html
+<template>
+  <div v-editable="blok"><!-- ... --></div>
+</template>
 ```
 
-See the jsfiddle to learn how to load the data from Storyblok:  
-https://jsfiddle.net/delooks/6tnttqvy/
+Where `blok` is the actual blok data coming from [Storblok's Content Delivery API](https://www.storyblok.com/docs/api/content-delivery?utm_source=github.com&utm_medium=readme&utm_campaign=storyblok-vue).
+
+Check out the [playground](/../../tree/next/playground) for a full example.
+
+### Compatibility
+
+This plugin is for Vue 3. Thus, it supports the [same browsers as Vue 3](https://github.com/vuejs/rfcs/blob/master/active-rfcs/0038-vue3-ie11-support.md). In short: all modern browsers, dropping IE support.
+
+## 🔗 Related Links
+
+- **[Add a headless CMS to Vue.js in 5 minutes](https://www.storyblok.com/tp/add-a-headless-CMS-to-vuejs-in-5-minutes?utm_source=github.com&utm_medium=readme&utm_campaign=storyblok-vue)**: Quick-start guide on getting up and running with Storyblok and Vue.
```

Comment:
@alexjoverm make it sense to link this here? Shouldn't we create Vue3 article? 

I would also mention Storyblok Vue/Nuxt Technology Hub.

Reply:
Very good points!